### PR TITLE
DOC: Improved the docstring of Series.any()

### PR DESCRIPTION
**Pull request by FrankDupree**
Checklist for the pandas documentation sprint (ignore this if you are doing
an unrelated PR):

- [x] PR title is "DOC: Improved the docstring of Series.any()"
- [ ] The validation script passes: `scripts/validate_docstrings.py Series.any()`
- [x] The PEP8 style check passes: `git diff upstream/master -u -- "*.py" | flake8 --diff`
- [x] The html version looks good: `python doc/make.py --single Series.any()`
- [x] It has been proofread on language by another sprint participant

Please include the output of the validation script below between the "```" ticks:

```
################################################################################
######################## Docstring (pandas.Series.any)  ########################
################################################################################

Return whether any element is True over requested axis.

Returns True if one (or more) elements are non-zero,
not-empty or not-False.

Parameters
----------
axis : {index (0)}
skipna : boolean, default True
    Exclude NA/null values. If an entire row/column is NA, the result
    will be NA.
level : int or level name, default None
    If the axis is a MultiIndex (hierarchical), count along a
    particular level, collapsing into a scalar.
bool_only : boolean, default None
    Include only boolean columns. If None, will attempt to use everything,
    then use only boolean data. Not implemented for Series.
**kwargs : Additional keywords have no effect but might be accepted for
    compatibility with numpy.

Returns
-------
any : scalar or Series (if level specified)

Examples
--------
>>> s1 = pd.Series([1, 2, 3])
>>> s1.any()
True

>>> import numpy as np
>>> s2 = pd.Series([np.NaN, np.NaN, np.NaN])
>>> s2.any()
False

################################################################################
################################## Validation ##################################
################################################################################

Errors found:
        Errors in parameters section
                Parameters {'kwargs'} not documented
                Unknown parameters {'**kwargs'}
                Parameter "axis" has no description
                Parameter "**kwargs" description should start with capital letter
        See Also section not found
```

Participants of the documentation sprint were told to go ahead with the description of the "**kwargs" parameter. 

Parameter "axis" has no description looks like a bug to me. Noticed some other methods show the same validation error. e.g "pandas.Series.mean" throws the same validation error.

### CODE REVIEW

**Comment by TomAugspurger** [comment]
Does this close a specific issue?

File: pandas/core/generic.py [hunk 1]
```diff
@@ -7784,25 +7784,49 @@ def _doc_parms(cls):
...
+    compatibility with numpy.
 
 Returns
 -------
-%(outname)s : %(name1)s or %(name2)s (if level specified)\n"""
+%(outname)s : %(name1)s or %(name2)s (if level specified)
+
+Examples
+--------
+>>> s1 = pd.Series([1,2,3])
```
Comment (TomAugspurger) [inline pandas/core/generic.py]: Could you add spaces after the commas here so this is pep8 compliant?

File: pandas/core/generic.py [hunk 2]
```diff
@@ -7784,25 +7784,49 @@ def _doc_parms(cls):
...
+>>> s1.any()
+True
+
+>>> import numpy as np
+>>> s2 = pd.Series([np.NaN,np.NaN,np.NaN])
+>>> s2.any()
+False
+
+>>> s3 = pd.Series([1,2,3,"Hobbit"])
```
Comment (TomAugspurger) [inline pandas/core/generic.py]: This last one is a bug: https://github.com/pandas-dev/pandas/issues/12863 It should return True / False.

I think just remove it for now.

Comment (FrankDupree) [reply pandas/core/generic.py]: yeah, I sensed it 👍

File: pandas/core/generic.py (hunk 1 above)
Comment (FrankDupree) [reply pandas/core/generic.py]: Okay, will do.

File: pandas/core/generic.py (hunk 2 above)
Comment (FrankDupree) [reply pandas/core/generic.py]: Done.

File: pandas/core/generic.py (hunk 1 above)
Comment (FrankDupree) [reply pandas/core/generic.py]: Done.

**Comment by datapythonista** [comment]
@FrankDupree can you please read carefully the doc on how to submit a PR [1], and make sure you do all the steps correctly? You surely missed the point about the title of the PR, and also copying the output of the `validate_docstrings.py` in the description. Without those, it's difficult to understand what you sent and review it.

Btw, in the description of the PR, the `[ ]` are checkboxes. You should make sure you completed every listed point, and mark them with an `[X]`, so we know you did.

Thank you!

1. https://python-sprints.github.io/pandas/guide/pandas_pr.html

**Comment by toobaz** [comment]
Are ``Parameter "axis" has no description`` and ``See Also section not found`` solved? If yes, could you update the description?

File: pandas/core/generic.py [hunk 3]
```diff
@@ -7784,25 +7784,41 @@ def _doc_parms(cls):
...
 _bool_doc = """
-
 %(desc)s
 
+Returns True if one (or more) elements are non-zero,
+not-empty or not-False.
+
 Parameters
 ----------
 axis : %(axis_descr)s
```
Comment (toobaz) [inline pandas/core/generic.py]: Missing description

**Review by toobaz — CHANGES_REQUESTED** [review]

File: pandas/core/generic.py (hunk 3 above)
Comment (toobaz) [reply pandas/core/generic.py]: (never mind)

**Comment by toobaz** [comment]
Please update the description with the validation script output

File: pandas/core/generic.py [hunk 4]
```diff
@@ -7784,25 +7784,41 @@ def _doc_parms(cls):
...
-    will be NA
+    will be NA.
 level : int or level name, default None
     If the axis is a MultiIndex (hierarchical), count along a
-    particular level, collapsing into a %(name1)s
+    particular level, collapsing into a %(name1)s.
 bool_only : boolean, default None
     Include only boolean columns. If None, will attempt to use everything,
     then use only boolean data. Not implemented for Series.
+**kwargs : Additional keywords have no effect but might be accepted for
```
Comment (jreback) [inline pandas/core/generic.py]: @datapythonista is this the language generally recommending here?

**Review by jreback — APPROVED** [review]

**Comment by TomAugspurger** [comment]
@FrankDupree Some things have changed in master. Can you check if there's anything else to be done?

File: pandas/core/generic.py (hunk 4 above)
Comment (datapythonista) [reply pandas/core/generic.py]: I didn't see this earlier. I think the description is similar to the ones used in `kwargs` with similar behavior. But it should go in the next line, as in the other parameters, and in this line simply leave `**kwargs`

**Comment by FrankDupree** [comment]
@TomAugspurger I will.

**Comment by jorisvandenbossche** [comment]
So it seems we had a conflicting PR https://github.com/pandas-dev/pandas/pull/20217 that implemented `DataFrame.any`, and since Series and DataFrame share code, this might have been duplicate work .. Sorry about that! 

I am going to close this PR, but please check if there is anything in your PR that is not in http://pandas-docs.github.io/pandas-docs-travis/generated/pandas.Series.any.html#pandas.Series.any. 
For example, I think it would be good to add an example of what an empty Series returns (Series([]).any()). Feel free to open a PR for that.

**Pull request closed without merging**